pkg/execution/svccatadmission/types.go

```diff
@@ -25,8 +28,17 @@ var (
 		Version:  sc_v1b1.SchemeGroupVersion.Version,
 		Resource: serviceBinding,
 	}
+	ingressResource = metav1.GroupVersionResource{
+		Group:    ext_v1b1.SchemeGroupVersion.Group,
+		Version:  ext_v1b1.SchemeGroupVersion.Version,
+		Resource: ingress,
+	}
```

Comment:
Move these three to `pkg/k8s` package too.

Reply:
fixed. Note that this refactoring added changes to files not related to this PR